include/swift/Basic/BlotMapVector.h

```diff
@@ -139,6 +139,6 @@ class BlotMapVector {
   SmallBlotMapVector() {}
 };
 
-} // end namespace swift
+} // namespace swift
```

Comment:
This is correct. Please do not remove these. See the namespace example in the LLVM style guide:

http://llvm.org/docs/CodingStandards.html#namespace-indentation